sensor-validate.c

```diff
@@ -1,28 +1,28 @@
 #include "sensor-validate.h"
 
-int _give_me_a_good_name(double value, double nextValue, double maxDelta) {
+int CheckSuddenJump(double value, double nextValue, double maxDelta) {
```

Comment:
`CheckSuddenJump` indicates that it will return 1 when sudden jump is there. But it is the other way round.